history/characters/wc_easteregg_characters.txt

```diff
@@ -457,4 +457,271 @@ easteregg13={ #Shrek reference
+easteregg17 = {
+	name = "Jubai"
+	#dna = jubai_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 4
+	martial = 22
+	stewardship = 4
+	intrigue = 3
+	learning = 3
+	prowess = 20
+	#Personality
+	trait = brave
+	trait = forgiving
+	trait = honest
+	#Education
+	trait = education_learning_4
+	#Other
+	trait = magic_good_3
+	506.3.10 = { 
+		birth = yes
+		trait = creature_tauren
+	}
+	583.1.1 = {
+		give_nickname = nick_the_ancient
+	}
+	# level 4 druid
+}
+
+# Dwarven officer of Ironforge
+easteregg18 = {
+	name = "Serphentos"
+	#dna = serphentos_dna
+	culture = bronzebeard
+	religion = khazism
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 3
+	martial = 20
+	stewardship = 8
+	intrigue = 5
+	learning = 3
+	prowess = 25
+	#Personality
+	trait = vengeful
+	trait = honest
+	trait = brave
+	#Education
+	trait = education_martial_4
+	#Other
+	trait = physique_good_2
+	trait = faith_warrior
+	trait = lifestyle_blademaster
+	377.8.24 = { 
+		birth = yes
+		trait = creature_dwarf
+	}
+	511.7.15 = {
+		employer = 17001 # Magni [4200]
+		give_council_position = councillor_marshal
+	}
+	# level 5 warrior
+}
+
+# Sentinel captain of the Alliance
+easteregg19 = {
+	name = "Monóri"
+	#dna = monori_dna
+	dynasty = easteregg_dyn11 # Silverleaf
+	culture = night_elf
+	religion = kaldorei_religion
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 10
+	martial = 15
+	stewardship = 5
+	intrigue = 3
+	learning = 4
+	prowess = 25
+	#Personality
+	trait = compassionate
+	trait = generous
+	trait = diligent
+	#Education
+	trait = education_martial_5
+	#Other
+	trait = beauty_good_3
+	trait = lifestyle_hunter
+	22.10.21 = { 
+		birth = yes
+		trait = creature_night_elf
+		effect = { add_character_flag = no_headgear }
+	}
+	507.1.1 = {
+		employer = 340008 # Malfurion [128003]
+		give_council_position = councillor_marshal
+	}
+	610.1.21 = { # Killed and risen as undead by Arthas
+		trait = being_undead
+	}
+	# level 5 huntress, later dark ranger
+}
+
+# Necromancer on the service of the Lich King who had infiltrated Stormwind
+easteregg20 = {
+	name = "Thainor"
+	#dna = thainor_dna
+	culture = lordaeronian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 5
+	martial = 5
+	stewardship = 4
+	intrigue = 10
+	learning = 10
+	prowess = 15
+	#Personality
+	trait = deceitful
+	trait = ambitious
+	trait = charming
+	#Education
+	trait = education_learning_4
+	#Other
+	trait = intellect_good_1
+	trait = magic_good_3
+	558.6.3 = { 
+		birth = yes
+		trait = creature_human
+	}
+	593.1.1={
+		employer = 10014		# Ner'zhul [2353]
+		religion = death_god
+	}
+	# level 4 necromantic
+}
+
+# A powerful and wise Tauren druid
+easteregg21 = {
+	name = "Gwaar"
+	#dna = gwaar_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 6
+	martial = 4
+	stewardship = 4
+	intrigue = 5
+	learning = 20
```

Comment:
20 base learning is massively overpowered.

Reply:
rebalanced